cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/components/src/preprocess.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
do we still need these preprocess.py files for images? if not, maybe can be removed